extensions/bond/ui/client/src/main/sawtooth/utils.cljs

```diff
@@ -63,7 +63,9 @@
         agent-contains #(re-find (re-pattern %) user-agent)]
         (cond (agent-contains "Firefox") :firefox
               (agent-contains "OPR") :opera
+              (or (agent-contains "MSIE") (agent-contains "Trident")) :ie
+              ; Order important: Edge lists Chrome/Safari, Chrome lists Safari
+              (agent-contains "Edge") :edge
               (agent-contains "Chrome") :chrome
               (agent-contains "Safari") :safari
-              (or (agent-contains "MSIE") (.-documentMode js/document)) :ie
               :default nil)))))
```

Comment:
Maybe. I like the semantics of `nil` though. `:unknown` feels somewhat arbitrary, while `nil`, as a falsey value, clearly communicates to me that the browser couldn't be found.